alerting/src/test/kotlin/org/opensearch/alerting/MonitorDataSourcesIT.kt

```diff
@@ -3151,7 +3151,7 @@ class MonitorDataSourcesIT : AlertingSingleNodeTestCase() {
         } catch (ex: java.lang.Exception) {
             exception = ex
         }
-        assertTrue(exception is java.util.NoSuchElementException)
+        assertFalse(exception is java.util.NoSuchElementException)
```

Comment:
Dont we want to move this into the catch and keep it as `assertTrue`?

Reply:
fixed it.